packages/lit-html/src/lit-html.ts

```diff
@@ -769,6 +790,14 @@ export class NodePart {
     }
   }
 
+  private _getTemplate(strings: TemplateStringsArray, result: TemplateResult) {
```

Comment:
Note this is an override point for ShadyDOM?

Reply:
Added.